packages/venia-concept/src/components/MiniCart/__stories__/MiniCart.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import 'src/index.css';
+import { Adapter } from 'src/drivers';
+import store from 'src/store';
+
+import MiniCart from '../miniCart';
+import defaultClasses from '../miniCart.css';
+
+const apiBase = new URL('/graphql', location.origin).toString();
+const noop = () => {};
+
+const nonConfigurableItem = {
+    __typename: 'SimpleProduct',
+    image: {
+        __typename: 'MediaGalleryEntry',
+        disabled: false,
+        file: '/v/a/va11-sg_main.jpg',
+        label: 'Main',
+        position: 1
+    },
+    item_id: 1,
```

Comment:
Since we don't regenerate this key React complains when we open storybook with multiple items rendered.

Reply:
Resolved in https://github.com/magento-research/pwa-studio/pull/1266/commits/4a15cba0b10219ed4f92d33b897cd2826944f93d.